Makefile

```diff
@@ -1,6 +1,7 @@
 SHELL := /bin/bash
 GITCOMMIT=$(shell git describe --tags HEAD)$(shell [[ $$(git status --porcelain) = "" ]] || echo -dirty)
 LDFLAGS="-X main.gitCommit=$(GITCOMMIT)"
+OPENSHIFT_INSTALL_DATA = ./vendor/github.com/openshift/installer/data/data	
```

Comment:
Where is this used? Also, can we use `:=` instead of `=`?

Reply:
This is override for data directory. Installer expects it to be in `./data/data`. I think in the binary mode this will not be a problem, just when running with `go run stuff`